ticker/src/test/java/com/robinhood/ticker/LevenshteinUtilsTest.java

```diff
@@ -0,0 +1,54 @@
+package com.robinhood.ticker;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class LevenshteinUtilsTest {
+    @Test
+    public void test_insert1() {
+        runTest("1111", "11211", "00100");
+    }
+
+    @Test
+    public void test_insert2() {
+        runTest("123", "0213", "0010");
+    }
+
+    @Test
+    public void test_delete() {
+        runTest("11211", "1111", "00200");
+    }
+
+    @Test
+    public void test_equal() {
+        runTest("1234", "1234", "0000");
+    }
+
+    @Test
+    public void test_completelyDifferent() {
+        runTest("1234", "5678", "0000");
+    }
+
+    @Test
+    public void test_shift() {
+        runTest("1234", "2345", "0000");
+    }
+
+    @Test
+    public void test_currency1() {
+        runTest("$123.99", "$1223.98", "00010000");
+    }
+
```

Comment:
I'd like to see a test with all three possible actions. something like "$123.45" -> "$1234.5" = "0000102"